Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build stage
+FROM node:14.11 as build-stage
+WORKDIR /app
+COPY package*.json ./
+RUN npm ci --silent
+COPY . .
+ENV NODE_ENV=production
+RUN npm run build
+
+# Production stage
+FROM node:14.11 as production-stage
```

Comment:
Thanks, you taught me something with the multi-stages build!
That's really cool :100:

Reply:
I'm glad I teached you something! I'd love a hacktober validation. Will get into the other comments you've pointed out as soon as I get off work 👍